src/index.tsx

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import { requireNativeComponent } from 'react-native';
 
-export interface OnChangeContext {
+export type OnChangeContext = {
   nativeEvent: {
     height: number;
     isFullScreen: boolean;
   };
-}
+};
 
-interface GripSize {
+type GripSize = {
   width: number;
   height: number;
-}
+};
 
-interface PullUpProps {
+type PullUpProps = {
```

Comment:
Is there a reason we switched it from `interface`s to `type`s?

Reply:
I just reverted the `index.js` to the master branch/iOS one, and created a separate `index.android.js`.

Reply:
👍